public/pages/CreateMonitor/containers/CreateMonitor/CreateMonitor.js

```diff
@@ -175,56 +1195,163 @@ export default class CreateMonitor extends Component {
       isDarkMode,
       notificationService,
     } = this.props;
-    const { createModalOpen, initialValues, plugins } = this.state;
+    const { createModalOpen, initialValues, plugins, pluginsLoading } = this.state;
+
     return (
       <div style={{ padding: '16px' }}>
         <Formik
+          innerRef={this.formikRef}
           initialValues={initialValues}
           onSubmit={this.evaluateSubmission}
           validateOnChange={false}
-          enableReinitialize={true}
+          enableReinitialize={false}
         >
-          {({ values, errors, handleSubmit, isSubmitting, isValid, touched }) => {
+          {({ values, errors, handleSubmit, isSubmitting, isValid, touched, setFieldValue }) => {
             const isComposite = values.monitor_type === MONITOR_TYPE.COMPOSITE_LEVEL;
+            const safeMonitor = this.buildMonitorForTriggers(values);
+            const safeTriggers = _.get(safeMonitor, 'triggers', []);
+
+            const ClassicToggleHeader = (
+              <EuiCheckbox
+                id="useClassicMonitorsHeader"
+                label={
+                  <span>
+                    Use classic monitors{' '}
```

Comment:
Looks like there are multiple `Use classic monitors` checkboxes in this file. Could you clarify their differences?